City-of-Helsinki/drupal-asuntotuotanto
# ASU-240 Inital configurations

How to test: 
- `make build post-install`
- Check that Gin -theme is installed and FI/SV/EN languages are enabled
